eloquentjs/8_1_retry.js

```diff
@@ -0,0 +1,37 @@
+class MultiplicationUnitFailure extends Error{};
+
+function primitiveMultiply(a, b) {
+    const innerPrimitiveMultiply = j => {
```

Comment:
When someone says "wrap a function" it means write a function that calls that function (either by being passed to it or called directly) and not to scope the function inside of the function. Try that pattern. Also, I'd look at the exercise again, you're not quite meeting the spec it gave.